package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stakhanov",
-  "version": "0.3.2",
+  "version": "0.4.0",
   "description": "AMQP worker library",
   "main": "index.js",
   "engines": {
```

Comment:
The node engine should specify the oldest supported node version, but should not enforce a specific one. It should be `"node": ">=6.0.0"`

Reply:
ok